### ISSUE TITLE
after require voyager artisan stops working

### ISSUE THREAD

**Issue by keradan** [thedevdojo/voyager#4658]
## Version information

 - Laravel: v6
 - PHP: 7.4
 - Database: MySQL 8.0
 - System: Mac OS mojave 10.14.6

## Description:
Trying to install voyager, I get some strange problems. After composer require tcg/voyager,  composer launches a script "@php artisan package:discover --ansi" and after that I wait for 300 seconds to see this error:

> [Symfony\Component\Process\Exception\ProcessTimedOutException]               
  The process "'/usr/local/Cellar/php/7.4.1/bin/php' -d allow_url_fopen='1' -  
  d disable_functions='' -d memory_limit='1536M' artisan package:discover --a  
  nsi" exceeded the timeout of 300 seconds.

After this error all attempts to call php artisan always return nothing in terminal, again long waiting and no information about errors or something else. Only after removing voyager package artisan starting to work again.

**Comment by MrCrayon** [comment]
Voyager version?

I can't see how this could depend from Voyager, maybe you are having connection problems and any large repository could timeout. 

You can try some of the solution listed in stackoverflow: 
https://stackoverflow.com/questions/18917768/why-composer-install-timeouts-after-300-seconds/18917919

https://stackoverflow.com/questions/28177019/composer-exceeded-the-timeout-of-300-seconds-while-installing-sylius-on-composer

Let us know if it works.

**Comment by keradan** [comment]
> Voyager version?

I try to install last voyager version, by "composer require tcg/voyager"

**Comment by keradan** [comment]
Maybe this will help - right after installing dependencies I see this:

> guzzlehttp/psr7 suggests installing zendframework/zend-httphandlerrunner (Emit PSR-7 responses)
intervention/image suggests installing ext-imagick (to use Imagick based image processing.)
intervention/image suggests installing intervention/imagecache (Caching extension for the Intervention Image library)

**Comment by MrCrayon** [comment]
Did you try any of the solutions in the links I posted?

**Comment by vkiranmaniya** [comment]
Please look at the your error saying <b>exceeded the timeout of 300 seconds.</b>, Maybe you are running on a developer machine having the slow processing power and that's why the installation requires more time to install the library. Meanwhile the <b>300 seconds</b> exceeds and operation fails itself. Or you may have a slow internet.

**Comment by MrCrayon** [comment]
Closed because no response.

**Comment by keradan** [comment]
> Closed because no response.

I finally figured out the problem is not in the Voyager, It was in the database connection. For some reason, when I trying to connect to the database using the default user - root without a password, the artisan freezes, all the logs were silent, and the script died after 300 seconds. I still don’t know what the reason is, so I work through a new user with password. And the voyager is a wonderful package, although the multilingual functionality did not suit me, but I chose another package for multilinguage.
